src/pages/instant-observability.js

```diff
@@ -110,11 +128,12 @@ const QuickstartsPage = ({ data, location }) => {
 
   const quickstarts = data.allQuickstarts.nodes;
 
-  const sortedQuickstarts = sortFeaturedPacks(quickstarts);
+  const sortedQuickstarts = sortFeaturedQuickstarts(quickstarts);
 
   const filteredQuickstarts = sortedQuickstarts
     .filter(filterBySearch(search))
-    .filter(filterByContentType(filter));
+    .filter(filterByContentType(filter))
+    .filter(filterByContentType(category));
```

Comment:
To keep things simple, let's break this filter out into a keywords/category function. That way the `filterByContentType` function doesn't have extra opaque functionality

Reply:
Sorry this function was designed to sort either content but it makes sense to split it out so it's easier to understand